src/uwtools/tests/drivers/test_fv3.py

```diff
@@ -219,38 +234,38 @@ def test_FV3_runscript(driverobj):
         assert [type(runscript.call_args.kwargs[x]) for x in args] == types
 
 
-def test_FV3__run_via_batch_submission(driverobj):
-    runscript = driverobj._runscript_path
-    with patch.object(driverobj, "provisioned_run_directory") as prd:
+def test_FV3__driver_config(driverobj):
+    assert driverobj._driver_config == driverobj._config["fv3"]
+
+
+def test_FV3__run_via_batch_submission(driverobj, truetask):
+    Path(driverobj._driver_config["execution"]["executable"]).touch()
+    with patch.object(driverobj, "provisioned_run_directory", truetask):
```

Comment:
Create a temp file representing the executable, and mock `provisioned_run_directory` with a task that always says its asset is ready.